webpack.config.js

```diff
@@ -4,6 +4,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 module.exports = {
   entry: {
     index: './src/index.js',
+    functionality: './src/functionality.js',
   },
```

Comment:
- [ ] When I ran your project I first got the warning below ⬇️  coming up even if your app worked as expected I would like to recommend you solve this issue by adding  **mode: 'development',** or **mode: 'production',** depending on which stage in your development process you are 👍  

![warming](https://user-images.githubusercontent.com/6615394/230503601-f12cfa64-8418-4e6a-8538-5c8bb61f5541.PNG)

Reply:
@codecaiine Thanks for pointing this out. I added the appropriate config in webpack.config.js.